src/shared-components/modal/index.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactModal from 'react-modal';
```

Comment:
as a follow up (not high priority), i'd like for us to refactor this modal to not use ReactModal. I don't think we need it and we're doing the immersive modal without it.

Reply:
Created an issue for follow up: https://github.com/PocketDerm/radiance-ui/issues/40#issue-403271409